hw/ip/rv_timer/data/rv_timer.hjson.tpl

```diff
@@ -131,7 +143,9 @@
         hwext: "true",
         hwqe: "true",
         fields: [
-          { bits: "0", name: "T", desc: "Interrupt test for timer" }
+          { bits: "0", name: "T", desc: "Interrupt test for timer",
```

Comment:
I thought this should be automatically excluded from the script. Have you seen any error related to this register?

Reply:
Hm I think this may still be around due to this being an old PR that I have rebased several times.

Let me remove these changes.

Reply:
Edit: these exclusions are apparently needed. The reason why these have not been present in this file (and why they show up as diffs now) is that they have only been added directly to the `hjson`, and not the `hjson.tpl` file.